common/amk/amk.h

```diff
@@ -0,0 +1,145 @@
+/**
+ * @file amk.h
+ * @author Cole Roberts (rober638@purdue.edu)
+ * @brief  Vroom
+ * @version 0.1
+ * @date 2024-10-11
+ *
+ * @copyright Copyright (c) 2024
+ *
+ */
+#ifndef _AMK_H_
+#define _AMK_H_
+
+#include <stdbool.h>
+#include <stdint.h>
+
+/* Inverter -> CAN */
+/* In AMK_Actual_Values_1 */
+typedef union
+{
+    struct {
+        uint16_t AMK_bReserve        : 8;
+        uint16_t AMK_bSystemReady    : 1;
+        uint16_t AMK_bError          : 1;
+        uint16_t AMK_bWarn           : 1;
+        uint16_t AMK_bQuitDcOn       : 1;
+        uint16_t AMK_bDcOn           : 1; /* Same as QUE ?? */
```

Comment:
Have you verified this yet?

Reply:
I am still looking into this. I am unsure if this is even necessary. We can charge the caps and monitor the DC bus voltage ourselves, correct? Or I would think once the precharging on our side is complete, we should be good to go. But that is a Finn question I would think.